builds/install/misc/replication.conf

```diff
@@ -6,6 +6,10 @@ database
 {
 	### ORIGIN SIDE SETTINGS
 
+	# Plugin used to perform replication.
+	#
+	# plugin = builtin
+
```

Comment:
I would prefer empty name for disabling replication. Now enabling it requires setting of log_directory that is pointless for external plugins (they have no access to Replication::Config).

Reply:
I agree with DS. In all other places when working with plugins we follow mentioned logic - empty plugins list disables feature, default is explicit. I see no problems with predefined name of plugin - we already have a lot in other places, starting with providers' names.

Reply:
Disabling is primarily done by excluding / commenting the database section of the replication.conf file. log_directory can be emptied to disable async replication, if required. Adding yet another level of enabled/disabled seems error-prone to me. But I'm not going to insist if others don't see that as over-complicated.

Reply:
> Disabling is primarily done by excluding / commenting the database section of the replication.conf file.

Actually it should be done by "ALTER DATABASE DISABLE PUBLICATION" at first hand and here comes another problem: I see no way to enable replication of all (any) database in the configuration file, only every one individually. Have I missed something?

Reply:
No, there's no such a way. Primarily because I cannot imagine multiple replicated databases sharing the same settings (first of all targets). While it's technically possible for them to share the log directories, it gonna be a maintenance nightmare for DBA. Perhaps we could match databases by patterns in replication.conf, but it requires serious thinking ( see above). Anyway, I'm open to ideas.

Reply:
Even if setting is set for a pattern it still can use individual settings for each matched database as in the trace config:
```
database = /db/(*.fdb)
{
  sync_replica = server2:/replicas/\1
}
```

Reply:
I'm afraid filename substitutions may be not enough. But I agree to consider that for v4.1.

Reply:
Is it too late to add field rdb$plugin_name into table rdb$publications?

Reply:
Instead of "plugin" in replication.conf?

Reply:
Yes. There is no point to have the same setting in two places. In this case active publication in database will be a sufficient for initializing replication from engine side and it can be automatically disabled later if/when internal plugin detect other required settings missing.

Reply:
Let's complete this PR with the config-based plugin setting. If adding a field would be possible after that (depends on whether some my changes will require ODS changes or not), it should be done with another PR.